File: Study/DataStructures/JavaCollections/src/Data Structures/Stacks.java
import java.util.Stack;

public class Stacks {
    public static void main(String[] args) {
        Stack<Integer> stack = new Stack<>();

        stack.push(10); // Push element
        stack.push(20);
        stack.push(30);
        
        System.out.println(stack.peek());  // Top element: 30
        System.out.println(stack.pop());   // Removes 30
        System.out.println(stack.isEmpty()); // Check if empty
    }
}

File: Study/DataStructures/C++ STL/Stacks.cpp
#include <iostream>
#include <stack>
using namespace std;

int main()
{
    // Initialisation
    stack<int> s;

    // Push
    s.push(32);

    // Pop
    s.pop();

    // Size
    int size = s.size();

    // Empty
    bool isEmpty = s.empty();

    // Top
    int top = s.top();

    // Printing the stack elements
    while (!s.empty())
        cout << s.top(), s.pop();
    return 0;
}